services/galley/src/Galley/API/Internal.hs

```diff
@@ -435,11 +435,14 @@ featureAPI =
     <@> mkNamedAPI getTeamSearchVisibilityInboundInternalMulti
   where
     fs g s = fsGet g <@> fsSet s
+    fsDeprecated g s = fsGet g <@> fsSetD s
```

Comment:
what's this about?

Reply:
ah, now i see the difference between the two definitions.  :)  if you don't want to add a comment, maybe it'll help future me if you change `fsSetD` to `fsSetDeprecated`?